main.js

```diff
@@ -860,7 +1038,7 @@ function updateSaveValuesFromForm(values) {
     values[87] = document.getElementById("sav-comedianstate").value;
     values[97] = document.getElementById("sav-papyrusstate").value;
     values[111] = document.getElementById("sav-shyrenstate").value;
-    values[231] = document.getElementById("sav-unkkills").value;
+    //values[231] = document.getElementById("sav-unkkills").value;
```

Comment:
?

Reply:
sav-unkkills no longer exists, per point 2.

Reply:
I read the description after I posted the review for some reason, but shouldn't it be removed completely then?

Reply:
That's fair.